### ISSUE TITLE
How to do batch write with block api?

### ISSUE THREAD

**Issue by lonelyleaf** [influxdata/influxdb-client-java#174]
__Proposal:__
batch write data with block api

__Use case:__
I need import data from file and batch write with block api,but I notice that although  `WriteApiBlocking.writePoints( String bucket, String org, List<Point> points)` accept collection of points,but the implementation of it write data one by one,which is too slow in my use case.

![batch-write](https://user-images.githubusercontent.com/9879800/98762643-15950b80-2413-11eb-9e57-c1bf85b233bf.png)

By the way, when I tried your non-blocking api,sometime it create too many `TIMED_WAITING` thread,and casue oom finally.

[threaddump-1604541290817.zip](https://github.com/influxdata/influxdb-client-java/files/5521962/threaddump-1604541290817.zip)

**Comment by bednar** [comment]
@lonelyleaf thanks, we will take a look

**Comment by bednar** [comment]
PR with fix is prepared #176. Meanwhile you could use this workaround:

```java
List<String> records = points.stream().map(Point::toLineProtocol).collect(Collectors.toList());

influxDBClient
    .getWriteApiBlocking()
    .writeRecords("b1", "org1", WritePrecision.NS, records);
```

**Comment by lonelyleaf** [comment]
@bednar thanks
